src/test/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployerTests.java

```diff
@@ -222,6 +235,51 @@ public void deployWithDeploymentServiceAccountNameDeploymentPropertyOverride() {
 		assertThat(podSpec.getServiceAccountName().equals("overridesan"));
 	}
 
+	@Test
+	public void deployWithGlobalTolerations() {
+		AppDefinition definition = new AppDefinition("app-test", null);
+
+		Map<String, String> props = new HashMap<>();
+		props.put("spring.cloud.deployer.kubernetes.tolerations.key", "test");
```

Comment:
one thing to note here -- they key would likely be say `spring.cloud.deployer.kubernetes.tolerations` and the value would be a JSON formatted string as with the volume mounts example. given these properties all go together, going this route keeps all config in one property rather than multiple

Reply:
Thanks for catching that, that is correct it would be a JSON Array value.  I have updated the tests.